phpstan.neon

```diff
@@ -24,3 +24,9 @@ parameters:
     # Symfony CLI parameter handling
     - message: '#^Cannot cast array<string>(\|bool)?\|string\|null to (int|string).$#'
       path: src/Command
+    # Doctrine dynamic method call error, findOneByXXX methods do not exist, but processed via _call
+    - message: '#^Call to an undefined method OAT\\SimpleRoster\\Repository\\UserRepository\:\:findOneByUsername\(\).$#'
+      path: src/Action/Security/RefreshJWTTokenAction.php
+    # Authentication doesn't seem to be Throwable to phpstan
+    - message: '#^Parameter \#1 \$exception of method OAT\\SimpleRoster\\Responder\\SerializerResponder\:\:createErrorJsonResponse\(\) expects Throwable, Symfony\\Component\\Security\\Core\\Exception\\AuthenticationException\|null given.$#'
```

Comment:
Yes, I agree with Andre

Reply:
Sorry, guys, but I don't see any other way to do this, after searching phpstan docs.
`AuthenticationException` extends `RuntimeException` which extends `\RuntimeException` which extends `Exception` which  is implementing goddamn `Throwable` interface. So, I don't see any way to change this except adding exclusion to phpstan rules.
p.s.
Ofc, this can be workaround with either changing underlying interfaces for Authentication providers (but who wants to put hands into framework core?) or by removing dependency from serializer responder, and manually preparing response somewhere in this class.
====
In any case - sorry, guys, but I'd prefer to leave this as is.
p.s
Conversation would be left opened